pandas/io/excel.py

```diff
@@ -1331,8 +1331,15 @@ def write_cells(self, cells, sheet_name=None, startrow=0, startcol=0,
             self.sheets[sheet_name] = wks
 
         if freeze_panes is not None:
-            wks.freeze_panes = wks.cell(row=freeze_panes[0] + 1,
-                                        column=freeze_panes[1] + 1)
+            if (
```

Comment:
Do you have any preference on whether _validate_freeze_panes is a method of the ExcelFile class or just a loose function in excel.py? Looking at other _validate functions in the code base it seems it can go either way.